src/main/content/_drafts/2018-07-19-installing-features-from-maven-dependencies.adoc

```diff
@@ -0,0 +1,183 @@
+---
+layout: post
+title: "Installing features from Maven dependencies in Open Liberty 18.0.0.2"
+date:   2018-07-19 11:30:00 -0000
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/29640130
+seo-title: Installing Liberty features from Maven dependencies - OpenLiberty.io
+seo-description: "Install features to the Open Liberty runtime kernel by adding dependencies in your Maven or Gradle build."
+blog_description: "Install features to the runtime kernel by adding dependencies in your Maven or Gradle build, which also provides Liberty API, SPI, and Java specification dependencies for compilation."
+---
+= Installing features from Maven dependencies in Open Liberty 18.0.0.2
+Eric Lau <https://github.com/ericglau>
+
+Open Liberty 18.0.0.2 is available in several runtime packages, including as a runtime kernel without any runtime features. With the latest releases of the Liberty https://github.com/WASdev/ci.maven[Maven] and https://github.com/WASdev/ci.gradle[Gradle] plugins, you can install runtime features simply by adding dependencies in your build. This also provides the relevant dependencies for compilation, so there is no longer any need to add Liberty API, SPI, or Java specification dependencies.
+
+== The runtime kernel
+
+If you're using Maven, here are the coordinates for the runtime kernel:
+[source,xml]
+----
+<assemblyArtifact>
```

Comment:
I guess we haven't done this, but I think the kernel should be a dependency as well. Perhaps a future improvement for the plugin.